src/plugins/intel_cpu/src/transformations/cpu_opset/common/op/read_value_with_subgraph.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/read_value.hpp"
+#include "openvino/op/util/sub_graph_base.hpp"
+#include "transformations/cpu_opset/common/op/submodel.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class ReadValueWithSubgraphNode : public op::util::MultiSubGraphOp {
+public:
+    OPENVINO_OP("ReadValueWithSubgraph", "cpu_plugin_opset");
+
+    ReadValueWithSubgraphNode();
+    ReadValueWithSubgraphNode(const std::shared_ptr<ov::op::util::Variable>& variable);
+
+    std::string get_variable_id() const;
+
+    void set_body(const std::shared_ptr<Model>& body) {
+        m_bodies[0] = body;
+    }
+    const std::shared_ptr<Model>& get_body() const {
+        return m_bodies[0];
```

Comment:
There is 'get_function()' method in the base class which does the same

Reply:
Done